inventory/host_vars/merger01.internal.opentechsjc.bonnyci.org

```diff
@@ -0,0 +1,3 @@
+---
+zuul_components:
```

Comment:
Moving this from group_vars to host_vars makes it easy to accidentally deploy a new zuul server instead of a new zuul merger when trying to add additional mergers.